certora/specs/AbdicatedFunctions.spec

```diff
@@ -11,18 +11,9 @@ methods {
     function Utils.toBytes4(bytes) external returns bytes4 envfree;
 }
 
-// Check that abdicating a function set their timelock to infinity.
-rule abidcatedFunctionHasInfiniteTimelock(env e, bytes4 selector) {
-    abdicateSubmit(e, selector);
-
-    assert timelock(selector) == max_uint256;
-}
-
-// Check that infinite timelocks can't be changed.
-rule inifiniteTimelockCantBeChanged(env e, method f, calldataarg data, bytes4 selector) {
```

Comment:
this is still true right?

Reply:
they are not explicitly prevented from being changed so I think if you're in a state where decrease timelock is scheduled for now, you can do it